CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [4.1.0] - 2020-10-28
+
+### Changed
+
+- Migrate to TypeScript ([#27](https://github.com/MetaMask/obs-store/pull/27))
+
+[Unreleased]:https://github.com/MetaMask/obs-store/compare/v4.1.0...HEAD
+[4.1.0]:https://github.com/MetaMask/obs-store/compare/v4.0.3...v4.1.0
```

Comment:
Sadly there are no tags in this repo, so these links don't work

Reply:
Ah yes, I was going to create those

Reply:
I created the tags that are linked